backend/medtagger/repositories/users.py

```diff
@@ -43,3 +43,9 @@ def set_user_info(user: User, firstName: str, lastName: str) -> None:
             user.first_name = firstName
             user.last_name = lastName
             session.add(user)
+
+    @staticmethod
+    def set_user_settings(settings: UserSettings) -> None:
```

Comment:
I would allow users to update only one key at a time here. It won't be so common to update all Settings at once.

This function may be helpful: https://docs.python.org/3.6/library/functions.html#setattr